dynawo/sources/Models/Modelica/Dynawo/Electrical/HVDC/BaseClasses.mo

```diff
@@ -37,14 +37,15 @@ package BaseClasses
     parameter Real KLosses "Losses coefficient between 0 and 1 : 1 if no loss in the HVDC link, < 1 otherwise";
     parameter Types.ActivePowerPu PMaxPu "Maximum active power in pu (base SnRef) flowing through the HVDC link";
 
-    input Types.ActivePowerPu P1RefPu(start = s10Pu.re) "Active power regulation set point in pu (base SnRef) at terminal 1 (receptor convention)";
+    input Types.ActivePowerPu P1RefPu(start = P1RefSetPu) "Active power regulation set point in pu (base SnRef) at terminal 1 (receptor convention)";
```

Comment:
In fact, I am not even sure this is needed here. I think we could keep this model exactly as it is and only change the _INIT models